layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
For reference, the following is from a 1.26-rc0 build on the edison:

root@edison:~# fdisk -l /dev/mmcblk0p7
Disk /dev/mmcblk0p7: 40 MiB, 41943040 bytes, 81920 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x00000000

and

root@edison:~# df -h /mnt/boot/                                                                                                                                                                                                              
Filesystem      Size  Used Avail Use% Mounted on
/dev/mmcblk0p7  6.1M  5.6M  574K  91% /mnt/boot